projects/storefrontapp-e2e-cypress/cypress/integration/regression/cart/added-to-cart-toast.e2e-spec.ts

```diff
@@ -15,11 +15,55 @@ describe('Added to cart toast - Anonymous user', () => {
       });
     });
 
-    it('should add product to cart', () => {
+    it('should add a product from a PDP to cart', () => {
+      const quantity = 4;
+
       cy.visit(`/product/${productId}`);
```

Comment:
Let's make the test more robust. Instead of going to a specific product, let's go to homepage, or a plp, click on the first product that appears. From there, retrieve the product Id and proceed with the test.